packages/flutter/lib/src/material/page.dart

```diff
@@ -67,21 +67,17 @@ class MaterialPageRoute<T> extends PageRoute<T> {
     @required this.builder,
     RouteSettings settings: const RouteSettings(),
     this.maintainState: true,
-    this.fullscreenDialog: false,
-  }) : super(settings: settings) {
-    assert(builder != null);
-    assert(opaque);
-  }
+    bool fullscreenDialog: false,
+  }) :
+      _fullscreenDialog = fullscreenDialog,
```

Comment:
this is pretty nit but I think all our other indents are 2 or 4 characters more than where the parent line starts. This will be the only case that makes children lines line up with where the parent line ends (in this case 5 characters from where the parent line starts).